data/anaconda.conf

```diff
@@ -272,6 +272,21 @@ can_change_root = False
 # Should the UI allow to change the configured users?
 can_change_users = False
 
+# Define the default password policies.
+# Specify a policy name and its attributes on each line.
+#
+# Valid attributes:
+#
+#   quality <NUMBER>  The minimal quality of the password.
```

Comment:
I think there should be a note about what "qaility" means - eq. that it's the password quality score defined by `libpwquality`.

Reply:
Fixed.